src/WebUI/Common/KubeSummary.tsx

```diff
@@ -269,7 +274,8 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
             className: "content-main-heading",
             description: this.props.clusterName
                 ? localeFormat(Resources.SummaryHeaderSubTextFormat, this.props.clusterName)
-                : localeFormat(Resources.NamespaceHeadingText, this.state.namespace || "")
+                : localeFormat(Resources.NamespaceHeadingText, this.state.namespace || ""),
+            commandBarItems: this.props.summaryPageHeaderCommandItems
```

Comment:
are we okay if this prop is null/undefined? i am assuming commandBarItems will check for it.

Reply:
yes, the prop type takes undefined, so they must be handling